content/learn/quick-start/getting-started/setup.md

```diff
@@ -130,17 +130,34 @@ It's not uncommon for debug builds using the default configuration to take multi
 Fortunately, there is a simple fix, and we don't have to give up our fast iterative compiles! Add the following to your `Cargo.toml`:
 
 ```toml
-# Enable a small amount of optimization in debug mode
+# Enable a small amount of optimization in debug mode.
 [profile.dev]
 opt-level = 1
 
-# Enable high optimizations for dependencies (incl. Bevy), but not for our code:
+# Enable a large amount of optimization in debug mode for dependencies.
 [profile.dev.package."*"]
 opt-level = 3
 ```
 
 You might think to simply develop in release mode instead, but we recommend against this as it can worsen the development experience by slowing down recompiles and disabling helpful debug symbols and assertions.
 
+In fact, you may want to optimize your release builds even further by adding the following to your `Cargo.toml`:
```

Comment:
The "In fact" sounds like it has something to do with the last sentence, which it does not. Maybe "When actually building in release mode to ship your game"?

Reply:
It is meant to be related to the previous sentence, like "not only are compile times worse in release mode, but we actually want to make the compile times _even worse_ for better performance".

Reply:
```suggestion
In fact, you may want to trade even more compile time in your release builds for performance by adding the following to your `Cargo.toml`:
```